Automattic/wp-calypso
# Color Schemes: Add Aquatic and hide Ocean

This is a phase one of https://github.com/Automattic/wp-calypso/issues/47256#issuecomment-724723856. Please refer to the plan to understand the scope of this PR.

#### Changes proposed in this Pull Request

* add aquatic color scheme (uses exact same colors as ocean)
* add temporary code in the color scheme picker to pretend "ocean" is "aquatic"

I know what's the usual reaction when people see _temporary code_. I'll personally make sure it will be followed upon :) 

#### Testing instructions

* set your account to use Ocean (either on local master branch or in prod https://wordpress.com/me/account)
* boot this PR
* confirm you can still see dark blue colors of "ocean" in the local UI
* navigate to `/me/account` on this branch and confirm the UI shows "aquatic" as the active scheme

Phase one of #47256
